propka/atom.py

```diff
@@ -1,66 +1,75 @@
-
-from __future__ import division
-from __future__ import print_function
-
-import string, propka.lib, propka.group
-
+"""Atom class - contains all atom information found in the PDB file"""
+import string
+import propka.lib
+import propka.group
+# TODO - what is hybrid36?
```

Comment:
Alternative PDB format that can transparently encode larger atom numbers. http://cci.lbl.gov/hybrid_36/

Reply:
Addressed in https://github.com/Electrostatics/propka-3.1/commit/66e3e552c970bd746b14920dc3949c5bf1b329b0